staging/src/k8s.io/component-base/logs/json/json.go

```diff
@@ -46,6 +47,48 @@ type zapLogger struct {
 // implement logr.Logger
 var _ logr.Logger = &zapLogger{}
 
+// getCallerFrame gets caller frame. The argument skip is the number of stack
```

Comment:
Best if functions are order the call stack, as those are called by Info, dbPanic and Error, they should be located after them.